src/reference/changes.rst

```diff
@@ -37,6 +37,89 @@ Cylc 8.5
    :rose: `2.4 <https://github.com/metomi/rose/blob/master/CHANGES.md>`__
 
 
+Triggering Multiple Tasks
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Cylc now respects the dependencies between tasks when triggering multiple
+tasks at the same time. This provides an easy way to re-run a group of tasks
+in order:
+
+.. image:: ../reference/changes/group-trigger.gif
+   :align: center
+   :width: 65%
+
+|
+
+This is generally easier than :ref:`using a new flow <interventions.reflow>`.
+
+Technical details:
+
+* Prerequisites on any tasks that are outside of the group of tasks being
+  triggered are automatically satisfied.
+* Any tasks which have already run within the group will be automatically
+  removed (i.e. ``cylc remove``) to allow them to be re-run without
+  intervention.
+* Any preparing, submitted or running tasks within the group will also be
+  removed if necessary to allow the tasks to re-run in order.
+
+
+Warning Triangles
+^^^^^^^^^^^^^^^^^
+
+The GUI now has warning triangles, these illuminate whenever warnings occur in
+the workflow.
+
+* Cylc logs warnings for various things such as task failures, stalled
+  workflows, and command errors. You can see these in the workflow log files.
+* When warnings occur, the warning triangle will illuminate for the
+  corresponding workflow.
+* Hover over the icon to reveal the warning.
+* Click on the icon to dismiss the warning.
+* If workflows are installed hierarchically, (e.g. the "development" and
+  "live" groups in this example), warnings will trickle up the hierarchy
+  to make it easier to monitor groups of workflows.
+* A log of all warnings can be found on the Dashboard page (last 10 warnings
+  for each workflow).
+
+.. image:: ../reference/changes/warning-triangles.gif
+   :align: center
+   :width: 95%
+
+|
+
+
+"Ghost" Tasks
+^^^^^^^^^^^^^
+
+The GUI and Tui now present some tasks in grey:
+
+.. image:: ../reference/changes/ghost-tasks.png
+   :align: center
+   :width: 65%
+
+|
+
+These are "ghost" tasks, they indicate something is that isn't presently there:
+
+* Waiting tasks that the scheduler hasn't moved onto yet (i.e.
+  :term:`inactive tasks <active task>` that are waiting).
```

Comment:
"Tasks that the scheduler hasn't moved onto yet" is not bad but sounds a bit clunky.  I like the idea of saying tasks in the "future graph", or even just "future tasks", for that. IMO that is pretty clear and intuitive, in relation to the active window.